### PR TITLE
Feature/admin organization management

### PR DESCRIPTION
**Pull request by o-zaengle**
adds ability to add owner to organization.
adds more tests and cleans up existing infrastructure around organization management.

### CODE REVIEW

File: Modules/Organization/Http/Handlers/Admin/Frontend/CreateOrganizationOwnerHandler.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class CreateOrganizationOwnerHandler extends AbstractHandler
+{
+    protected $organizationRepository;
+
+    public function __construct(OrganizationRepository $repository)
+    {
+        $this->organizationRepository = $repository;
+    }
+
+    public function listenTo(Request $request)
```
Comment (jesseschutt) [inline Modules/Organization/Http/Handlers/Admin/Frontend/CreateOrganizationOwnerHandler.php]: Let's get the docblocks in place.

File: Modules/Organization/Http/admin-routes.php [hunk 2]
```diff
@@ -21,5 +21,16 @@
         Route::get('/', 'OrganizationController@show')->name('show');
         Route::delete('/', 'OrganizationController@destroy')->name('destroy');
         Route::put('/', 'OrganizationController@update')->name('update');
+
+        Route::group([
+            'prefix' => 'owner',
```
Comment (jesseschutt) [inline Modules/Organization/Http/admin-routes.php]: I think this should be plural  "owners".

File: Modules/Organization/Tests/Feature/Admin/OrganizationOwnerTest.php [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    public function it_shows_create_owner_view_to_admins()
+    {
+        $response = $this->actingAs($this->admin)->get(route('admin.web.organization.owner.create', [
+            'organization' => $this->organization->getRouteKey(),
+        ]));
+
+        $response->assertViewIs('organization::admin.owner.create')
+            ->assertViewHas(['users']);
+
+        $users = $this->getResponseData($response, 'users');
```
Comment (jesseschutt) [inline Modules/Organization/Tests/Feature/Admin/OrganizationOwnerTest.php]: Good pickup on the `->getResponseData()` method!

File: Modules/Organization/Tests/Unit/Repositories/OrganizationRepositoryTest.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+        $this->assertNotEmpty($updated->getChanges());
+
+        $this->assertDatabaseHas('organizations', [
+            'uuid'     => $organization->uuid,
+            'owner_id' => null,
+        ]);
+    }
+
+    protected function setUp()
```
Comment (jesseschutt) [inline Modules/Organization/Tests/Unit/Repositories/OrganizationRepositoryTest.php]: Can we please keep the `setUp()` method at the top of the test file as it is acting in place of the `__construct`?

File: Modules/Organization/Http/admin-routes.php (hunk 2 above)
Comment (o-zaengle) [reply Modules/Organization/Http/admin-routes.php]: Changes have been made.

File: Modules/Organization/Http/Handlers/Admin/Frontend/CreateOrganizationOwnerHandler.php (hunk 1 above)
Comment (o-zaengle) [reply Modules/Organization/Http/Handlers/Admin/Frontend/CreateOrganizationOwnerHandler.php]: done

**Pull request merged**